### PR TITLE
Update app-templates.md

### PR DESCRIPTION
**Pull request by SurbhiJainMSFT**
Added Classroom Drop-in app template

### CODE REVIEW

File: msteams-platform/samples/app-templates.md [hunk 1]
```diff
@@ -100,6 +100,15 @@ Checklist is a custom Microsoft Teams [messaging extension](../messaging-extensi
 :::column-end:::
 :::row-end:::
 
+## Classroom Drop-in
+
+Classroom Drop-in is a Microsoft [Power Platform](https://powerapps.microsoft.com/blog/now-in-preview-customize-teams-with-built-in-power-platform-capabilities/)-based app that enables system leaders to find class teams (virtual classrooms) and add themselves or others to these class teams for a specified drop-in period, as needed. The app, built using Microsoft [Power Apps](/powerapps/powerapps-overview), and [Power Automate](/power-automate/getting-started), deeply integrates with Microsoft Teams and enables educational institutes to ensure relevant stakeholders have access to class teams for required periods of time to ensure smooth operations in a hybrid learning environment.
+
+
```
Comment (KirtiPereira) [inline msteams-platform/samples/app-templates.md:107]:
```Suggestion
Classroom Drop-in is a Microsoft [Power Platform](https://powerapps.microsoft.com/blog/now-in-preview-customize-teams-with-built-in-power-platform-capabilities/)-based app that enables system leaders to find class teams (virtual classrooms) and add themselves or others to these class teams for a specified drop-in period. The app built using Microsoft [Power Apps](/powerapps/powerapps-overview) and [Power Automate](/power-automate/getting-started), deeply integrates with Microsoft Teams. This enables educational institutes to ensure relevant stakeholders have access to class teams for required period of time, and smooth operations in a hybrid learning environment.
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
Hi @SurbhiJainMSFT  - please verify and commit the suggested changes.

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
Please commit the changes.
Thank you.

**Review by KirtiPereira — APPROVED** [review]
Approved for merging.

**Review by surbhigupta12 — APPROVED** [review]
Approving the file.

**Pull request merged**